tests/magma/compiler/test_compiler.py

```diff
@@ -262,6 +267,32 @@ def test_find_process_circular(self):
         self.assertEqual(set(procs4), all_procs)
         self.assertEqual(set(procs6), all_procs)
 
+    def test_find_process_ref_ports(self):
+        """Checks finding all processes for RefPort connection.
+        [p1 -> ref/var -> p2 -> in/out -> p3]"""
+
+        # Create processes
+        p1, p2, p3 = (
+            ProcA(),
+            ProcB(),
+            ProcC()
+        )
+        # Create complicated circular structure with joins and forks
+        p1.ref.connect(p2.var_port)
+        p2.out.connect(p3.inp)
+
+        # Regardless where we start searching...
```

Comment:
Did you just copy paste a unit test and is this still tentative or what is this test really about?

Reply:
Yes, the test was copied to keep a similar structure and I forgot to change a comment. It tests finding the connected processes if they are connected via a Ref/Var-port.